src/game_api/entity_db.hpp

```diff
@@ -86,7 +88,7 @@ struct EntityDB
     int32_t tile_y;
     uint8_t damage;
     uint8_t life;
-    uint8_t sacrifice_value;
+    uint8_t sacrifice_value; // Favor for sacrificing alive. Halved when dead (health == 0).
```

Comment:
And I believe the autodoc only considers triple comments on the previous lines.

Reply:
ah ok cool.